### DISCUSSION TITLE
A first look at the Interactivity API

### DISCUSSION THREAD

**Discussion by mburridge** [WordPress/developer-blog-content#141] [Topic Ideas]
The Interactivity API is coming. It was proposed in [this Make post](https://make.wordpress.org/core/2023/03/30/proposal-the-interactivity-api-a-better-developer-experience-in-building-interactive-blocks/).

This article would be a "first look" at the Interactivity API and include a basic tutorial to showcase the possibilities of the new API.

**Comment by bph** [comment]
The Interactivity API is on the [Roadmap for WordPress 6.5](https://make.wordpress.org/core/2023/12/07/roadmap-to-6-5/) to be a public API. It's time to get this topic on the to-do-list for early 2024.

**Comment by bph** [comment]
Discussion move to the issues after approval. Please continue the conversation there

- https://github.com/WordPress/developer-blog-content/issues/188

**Comment by bph** [comment]
Published: [A first look at the Interactivity API](https://developer.wordpress.org/news/2024/04/11/a-first-look-at-the-interactivity-api/)
